packages/core/src/utils/JsonTransformer.ts

```diff
@@ -25,8 +51,12 @@ export class JsonTransformer {
     return JSON.stringify(this.toJSON(classInstance))
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static deserialize<T>(jsonString: string, Class: { new (...args: any[]): T }): T {
-    return this.fromJSON(JSON.parse(jsonString), Class)
+  public static deserialize<T>(
+    jsonString: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Class: { new (...args: any[]): T },
```

Comment:
```suggestion
    class: { new (...args: any[]): T },
```

Reply:
I assume you mean `cls`?

Reply:
`'class' is not allowed as a parameter name.ts(1390)`

Reply:
Anything other than pascalcase is fine by my :)